### DISCUSSION TITLE
Insert new panels at bottom of dashboard instead of top

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#38218] [Dashboards, Panels & Visualizations]
Generally, I build new dashboards from the top down, and I am guessing that most people do the same. And, if I am adding something to existing dashboard, I want the new panel at the bottom of the dashboard more often than at the top.

For these reasons, it would be helpful if the "Add Panel" button inserted the new panel at the bottom of the dashboard rather than the top.

**Comment by leeoniya** [comment]
i think it's a good idea, as this can also reduce the amount of layout re-adjustment you need to keep (un)doing as you add more panels.

one issue is what happens when you already have a screen full of panels? adding a new one will be super non-obvious, since it can be off-screen -- we'd need to scroll it into view.

**Comment by nabelekt** [comment]
Agreed; I forgot to mention that- I imagined that the dashboard would be scrolled to the bottom where the new panel is inserted.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by nabelekt** [comment]
Any updates here? Still a pretty big annoyance for me.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #83002. Feel free to continue the discussion around this there. Thank you!
